changelog.json

```
@@ -1,5 +1,15 @@
 {
   "releases": {
+    "2.5.0": [
+      "[New] Add, push, and view git tags in Desktop - #9424",
```

The intent here is just to increase discoverability a bit for people who want to use this but aren't sure where to look.
```suggestion
      "[New] Add, push, and view Git tags on commit history in Desktop - #9424",
```